### ISSUE TITLE
1.1.0.0+ has an orphan BEX dependency 

### ISSUE THREAD

**Issue by Kornstalx** [NBKRedSpy/BtShowXp#1]
1.0.4.2 is the last release of this mod that will work with vanilla.
1.1.0.0+ fails to load in modtek and kicks out dependency errors in the log (it's looking for Extended_CE)

```
Expanding manifest BtShowXp:
	:StreamingAssets
	Loaded assembly BtShowXp (v1.3.0.0)
	Invoking 'HarmonyInit.Init(String directory, String settingsJSON)' using parameter dictionary
	Error: While invoking 'HarmonyInit.Init', an exception occured
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeLoadException: Could not load type of field 'BtShowXp.Core:_bexModSettings' (5) due to: Could not load file or assembly 'Extended_CE, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. assembly:Extended_CE, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null type:<unknown type> member:(null) signature:<none>
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <d7ac571ca2d04b2f981d0d886fa067cf>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <d7ac571ca2d04b2f981d0d886fa067cf>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <d7ac571ca2d04b2f981d0d886fa067cf>:0 
  at ModTek.Util.AssemblyUtil.InvokeMethodByParameterNames (System.Reflection.MethodInfo method, System.Collections.Generic.Dictionary`2[TKey,TValue] paramsDictionary) [0x0014c] in <79fdd5cba23f4f27a360615c6abad375>:0 
  at ModTek.ModTek.LoadAssemblyAndCallInit (ModTek.ModDefEx modDef) [0x0019c] in <79fdd5cba23f4f27a360615c6abad375>:0 
```

**Comment by NBKRedSpy** [comment]
Thank you for reporting this issue, and especially taking the time to look if earlier versions work.

I've installed a clean copy of BEX with the latest CAB and BTShowXp and I cannot reproduce.

A couple of questions:

- Is the BEX version from Extended_CE_1_9_3_7.zip?
- Are there any other mods added to the base install?  

If you would like to test against a base install without affecting the current install, rename Mods to Mods.Original, create a new empty Mods folder and install BEX again with the instructions.

Thanks again for taking the time to report the issue.

**Comment by NBKRedSpy** [comment]
Deleted the previous comment.  I missed that you are talking about Vanilla.  Looking into it now. 

Thank you for reporting.

**Comment by NBKRedSpy** [comment]
Fixed in [1.3.1.0](https://github.com/NBKRedSpy/BtShowXp/tree/1.3.1.0). 

Thank you for reporting that!
